web/ui/react-app/package.json

```diff
@@ -16,9 +16,9 @@
     "@codemirror/state": "^0.19.6",
     "@codemirror/view": "^0.19.27",
     "@forevolve/bootstrap-dark": "^1.0.0",
-    "@fortawesome/fontawesome-svg-core": "^1.2.36",
-    "@fortawesome/free-solid-svg-icons": "^5.7.2",
-    "@fortawesome/react-fontawesome": "^0.1.16",
+    "@fortawesome/fontawesome-svg-core": "1.2.36",
+    "@fortawesome/free-solid-svg-icons": "5.15.4",
+    "@fortawesome/react-fontawesome": "0.1.16",
```

Comment:
pinning this release since 0.1.17 will cause build failures as it works with FA 6.0

Reply:
I see that we can upgrade it to 6.0 easily, some icons will look a bit different but no breaking changes, especially that there aren't that many icons used anyway

Reply:
Yeah, sounds good to me if you want to try upgrading to 6.0.